File: HackerRank/Interview Preparation Kit/Trees/height_of_a_binary_tree.c
/* you only have to complete the function given below.  
node is defined as  

struct node {
    
    int data;
    struct node *left;
    struct node *right;
  
};

*/
int getHeight(struct node* root) {
    if(root == NULL){
        return -1;
    }else {
        int lheight = getHeight(root -> left);
        int rheight = getHeight(root -> right);
        
        if(lheight > rheight){
            return (lheight + 1);
        } else{
            return (rheight + 1);
        }
    }   
}

File: HackerRank/Interview Preparation Kit/Trees/HeightOfABinaryTree.java
	/*
    class Node 
    	int data;
    	Node left;
    	Node right;
	*/
	static int getHeight(Node root) {
      	// Write your code here.
        if(root == null)
            return -1;
        return Math.max(getHeight(root.left), getHeight(root.right)) + 1;
    }